examples/todo-list/src/repositories/todo-list-image.repository.ts

```diff
@@ -39,23 +34,21 @@ export class TodoListImageRepository extends DefaultCrudRepository<
       'todoList',
       todoListRepositoryGetter,
     );
+    this.registerInclusion('todoList', this.todoList.inclusionResolver);
   }
 
-  async find(
+  protected async includeRelatedModels(
+    entities: TodoListImage[],
     filter?: Filter<TodoListImage>,
-    options?: Options,
-  ): Promise<TodoListImageWithRelations[]> {
-    // Prevent juggler for applying "include" filter
-    // Juggler is not aware of LB4 relations
-    const include = filter && filter.include;
-    filter = {...filter, include: undefined};
-
-    const result = await super.find(filter, options);
+    _options?: Options,
+  ): Promise<(TodoListImage & TodoListImageRelations)[]> {
+    const result = entities as (TodoListImage & TodoListImageRelations)[];
```

Comment:
Why did you change `TodoListImageWithRelations[]` to `(TodoListImage & TodoListImageRelations)[]`?

Reply:
That was not intentional. Thanks for spotting this change, I am going to revert this code back to use `TodoListImageWithRelations`.

Reply:
In fact, `includeRelatedModels` should not be needed now that the constructor calls `this.registerInclusion('todoList', this.todoList.inclusionResolver)`